cognitedata/reveal
# feat(react-components): hooks to support point cloud annotation assets

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/BND3D-2642

## Description :pencil:

Enables a hook to get all mapped assets (annotation) for given point cloud model.

## How has this been tested? :mag:

In Storybook


## Test instructions :information_source:

- `yarn storybook`
- Load `SearchHooks` example
- Select `All asset search mappings`
- It should populate all the asset mapped to `CAD`, `Point Cloud` and `360 Image` assets in the grid
- Search `Process Piping` in the input field and click on `Search` Button


## Checklist :ballot_box_with_check:
